packages/schemas/src/seeds/tenant.ts

```diff
@@ -5,4 +5,13 @@ import type { Tenants } from '../models/tenants.js';
 export const defaultTenantId = 'default';
 export const adminTenantId = 'admin';
 
-export type TenantModel = InferModelType<typeof Tenants>;
+export enum TenantTag {
+  Development = 'development',
+  Staging = 'staging',
+  Production = 'production',
+}
+
+export type TenantModel = Pick<
+  InferModelType<typeof Tenants>,
+  'id' | 'dbUser' | 'dbUserPassword' | 'name'
+> & { tag: TenantTag; createdAt: number };
```

Comment:
For this case, should we use omit instead of pick?  This type, by design, should only replace the tag type using TenantTag and keep anything else the same.   I.e. if we introduce any new field in the Tenant table, no need to update this type?

Reply:
updated.